issuer-front-end/README.md

```diff
@@ -0,0 +1,100 @@
+# Issuer Front End Wallet Testing
+
+This front end is used for testing the new browser wallet functionalities concerning the web3ID infrastructure.
+You can manually issue/create credentials using this front end. Only the browser wallet (no walletConnect) is supported in the first version.
+The front end initializes and interacts with the [`credential_registry` smart contract](https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/examples/credential-registry).
+
+The available front-end flows are:
+
+Positive test cases:
+
+- 1. step to create issuer keys.
+- 2. step to deploy new `credential_registry` smart contract (each issuer deploys their own smart contract).
+- 3. step to input the newly created `credential_registry` smart contract index.
+- 4. step to issue a new credential.
+- 5. step to display the credential entry in the `credential_registry` smart contract.
+- 6. step to create a zero-knowledge proof about the credential.
+
+Negative test cases:
+
+- 7. step to issue a new credential (Issuer registers credential delayed in the smart contract).
```

Comment:
When adding a credential to the browserWallet with the `addWeb3IdCredentia( .... , callbackFunction)` entrypoint, you provide a `callbackFunction`. This `callbackFunction` would usually call the backend and wait until the backend registered the credential in the smart contract and after that returns a `signature/randomness/proof` on the commitments to the front end. This means the intended flow would be that the browser wallet completes the `addWeb3IdCredentia( .... , callbackFunction)` entrypoint successfully only when the credential is also registered in the smart contract successfully. 
If a user clicks on the "verifiable Credential" section (in the below screenshot) after the completion of the `addWeb3IdCredentia()` flow, it would see its credential since the credential is registered successfully in the smart contract.

Since we don't have control over the correctness of this backend when third-party run their own backend. There could be two negative scenarios:
-  The backend does not successfully register the credential in the smart contract but still returns the correct `signature/randomness/proof` on the commitments to the front end.
-  The backend does successfully register the credential in the smart contract but does not wait until it happens and immediately returns the correct `signature/randomness/proof` on the commitments to the front end.

If a user clicks on the "verifiable Credential" section (in the below screenshot) after the completion of the `addWeb3IdCredentia()` in the two negative scenarios, it would NOT see its credential since the credential is NOT registered successfully in the smart contract. In the second case, it would appear after the registration in the smart contact (because it is delayer) has been completed.

![Screenshot from 2023-08-11 13-52-14](https://github.com/Concordium/concordium-web3id/assets/13039169/f9474176-2867-41d6-a397-5aebd19187f1)

Reply:
Thanks for the explanation. Can we have some of this explanation on the page itself though?